client/src/test/java/io/pravega/client/segment/impl/SegmentOutputStreamTest.java

```diff
@@ -217,7 +248,7 @@ public void testNewEventsGoAfterInflight() throws ConnectionFailedException {
         cf.provideConnection(uri, connection);
         @SuppressWarnings("resource")
         SegmentOutputStreamImpl output = new SegmentOutputStreamImpl(SEGMENT, controller, cf, cid, segmentSealedCallback, RETRY_SCHEDULE, "");
-        
```

Comment:
Although it is great that you are fixing these spurious spaces, I'd rather in general do such formatting changes in separate a PR. Leave it for this PR because it is not enough to confuse the reviewer, but in general, let's avoid them, please.

Reply:
I think this is unintentional due to the IDE auto-formatting. I have the same issue and I hate to manually go back and unfix all the auto-fixes.